.github/workflows/code-quality.yml

```diff
@@ -1,57 +0,0 @@
-name: Code Quality
```

Comment:
Do we want to remove this?

Reply:
Yeah, i merged it into `Solidity` since that's the only place that uses these checks now